bundles/org.openhab.binding.loxone/src/main/java/org/openhab/binding/loxone/internal/controls/LxControlJalousie.java

```diff
@@ -131,8 +131,15 @@ public void initialize(LxControlConfig config) {
     }
 
     private void handleOperateCommands(Command command) throws IOException {
+        logger.debug("Command input {}", command);
         if (command instanceof PercentType) {
-            moveToPosition(((PercentType) command).doubleValue() / 100);
+            if (((PercentType) command).compareTo(PercentType.ZERO) == 0) {
```

Comment:
A equals should work:
```suggestion
            if (PercentType.ZERO.equals(command)) {
```
Same below